deployability/modules/workflow_engine/task.py

```diff
@@ -45,7 +49,11 @@ def execute(self) -> None:
                     task_args.extend([f"--{key}={argvalue}" for argvalue in value])
                 else:
                     task_args.append(f"--{key}={value}")
-        self.logger.debug(f'Running task "{self.task_name}" with arguments: {task_args}')
+            else:
+                logger.error(f'Coulg not parse arguments {arg}')
```

Comment:
Typo: Coul`g` -> Coul`d`

Reply:
Done in [ebce158](https://github.com/wazuh/wazuh-qa/pull/4960/commits/ebce158066ecff2dc0cb82cc7f902dc54b6d489d)